changenow-funding-py/src/agent.py

```diff
@@ -56,33 +61,37 @@ def detect_changenow_funding(w3, transaction_event):
     global LOW_VOL_ALERT_COUNT
     global NEW_EOA_ALERT_COUNT
     global DENOMINATOR_COUNT
+    global CHAIN_ID
+
+    changenow_threshold = CHANGENOW_THRESHOLD[CHAIN_ID]
+    changenow_addresses = CHANGENOW_ADDRESSES[CHAIN_ID]
 
     findings = []
-    chain_id = w3.eth.chain_id
+
     native_value = transaction_event.transaction.value / 10e17
 
     # logging.info(f"Analyzing transaction {transaction_event.transaction.hash} on chain {chain_id}")
 
-    logging.info(f"Value: {transaction_event.transaction.value}")
-    logging.info(f"Transaction value: {native_value}")
-    logging.info(f"Address transaction count: {w3.eth.get_transaction_count(Web3.toChecksumAddress(transaction_event.to))}")
-    if (native_value > 0 and (native_value < CHANGENOW_THRESHOLD[chain_id] or is_new_account(w3, transaction_event.to)) and not is_contract(w3, transaction_event.to)):
+    # logging.info(f"Value: {transaction_event.transaction.value}")
+    # logging.info(f"Transaction value: {native_value}")
+    # logging.info(f"Address transaction count: {w3.eth.get_transaction_count(Web3.toChecksumAddress(transaction_event.to))}")
```

Comment:
i would recommend to remove as opposed to commenting out

Reply:
sure makes sense 👍